migtests/tests/pg/omnibus/sql/00_create_users_roles.ddl.sql

```diff
@@ -0,0 +1,28 @@
+-- running with role postgres with admin privileges
```

Comment:
General comment: For the Schema, see if we can use the zipped file and unzip it in the script to reduce the size/number of files in the repo